cli/command/run/espresso.go

```diff
@@ -64,8 +65,14 @@ func NewEspressoCmd() *cobra.Command {
 	f.StringVar(&lflags.TestOptions.Package, "testOptions.package", "", "Include package")
 	f.StringVar(&lflags.TestOptions.Size, "testOptions.size", "", "Include tests based on size")
 	f.StringVar(&lflags.TestOptions.Annotation, "testOptions.annotation", "", "Include tests based on the annotation")
-	f.IntVar(&lflags.TestOptions.ShardIndex, "testOptions.shardIndex", 0, "The shard index for this particular run")
-	f.IntVar(&lflags.TestOptions.NumShards, "testOptions.numShards", 0, "Total number of shards")
+	var shardIndex, numShards int
+	f.IntVar(&numShards, "testOptions.numShards", 0, "Total number of shards")
+	f.IntVar(&shardIndex, "testOptions.shardIndex", -1, "The shard index for this particular run")
```

Comment:
I'd recommend to completely remove the ability to specify the `shardIndex`. Just have the user specify `numShards`, or rather`shards` since `numShards` is somewhat redudant at that point and handle the rest automatically. Easier to explain to the user how this works and also less edge cases to tackle on our end.

Reply:
That definitely simplifies things. 👍